e2etest/local_transfer_test.go

```diff
@@ -683,8 +672,8 @@ func TestChainIDWithJutland(t *testing.T) {
 		require.NoError(t, err)
 		err = bc.CommitBlock(blk)
 		require.NoError(t, err)
-		require.Equal(t, c.success, len(blk.Actions))
-		require.Equal(t, c.success, len(blk.Receipts))
+		require.Equal(t, 1, len(blk.Actions))
```

Comment:
for the last test case {2, false}, it should be `len(blk.Actions) == 0` since the tx will be rejected?

compare to left side code, the logic has changed. We need to check/understand/fix what's wrong, not just change a value to let test pass?